docs/getting-started/dashboard_tutorial.md

```diff
@@ -0,0 +1,1788 @@
+
+```python eval
+rx.table.root(
+        rx.table.header(
+            rx.table.row(
+                rx.table.column_header_cell("Name"),
+                rx.table.column_header_cell("Email"),
+                rx.table.column_header_cell("Gender"),
+            ),
+        ),
+        rx.table.body(
+            rx.table.row(
+                rx.table.cell("Danilo Sousa"),
+                rx.table.cell("danilo@example.com"),
+                rx.table.cell("Male"),
+            ),
+            rx.table.row(
+                rx.table.cell("Zahra Ambessa"),
+                rx.table.cell("zahra@example.com"),
+                rx.table.cell("Female"),
+            ),
+        ),
+        variant="surface",
+        size="3",
+    )
+```
+
+```python
+def index() -> rx.Component:
+    return rx.table.root(
+        rx.table.header(
+            rx.table.row(
+                rx.table.column_header_cell("Name"),
+                rx.table.column_header_cell("Email"),
+                rx.table.column_header_cell("Gender"),
+            ),
+        ),
+        rx.table.body(
+            rx.table.row(
+                rx.table.cell("Danilo Sousa"),
+                rx.table.cell("danilo@example.com"),
+                rx.table.cell("Male"),
+            ),
+            rx.table.row(
+                rx.table.cell("Zahra Ambessa"),
+                rx.table.cell("zahra@example.com"),
+                rx.table.cell("Female"),
+            ),
+        ),
+        variant="surface",
+        size="3",
+    )
+```
+
+## Showing dynamic data (State)
+
+Up until this point all the data we are showing in the app is static. This is not very useful for a data dashboard. We need to be able to show dynamic data that can be added to and updated.
+
+This is where `State` comes in. `State` is a Python class that stores variables that can change when the app is running, as well as the functions that can change those variables.
+
+To define a state class, subclass `rx.State` and define fields that store the state of your app. The state variables (vars) should have a type annotation, and can be initialized with a default value. Check out the [basics]({docs.getting_started.basics.path}) section for a simple example of how state works.
+
+
+In the example below we define a `State` class called `State` that has a variable called `users` that is a list of lists. Each list in the `users` list represents a user and contains their name, email and gender.
+
+```python
+class State(rx.State):
+    users: list[list] = [
+        ["Danilo Sousa", "danilo@example.com", "Male"],
+        ["Zahra Ambessa", "zahra@example.com", "Female"],
+    ]
+```
+
+To iterate over a state var that is a list, we use the [`rx.foreach`]({docs.components.rendering_iterables.path}) function to render a list of components. The `rx.foreach` component takes an `iterable` (list, tuple or dict) and a `function` that renders each item in the `iterable`.
```

Comment:
As a new user, I wonder why I can't just splat a for loop here. Maybe a quick mention/redirect to that section of the basic docs?

Reply:
this is a fantastic point, I have added a link to the runtime vs compile time part of the basics section

Reply:
Nice!